src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="footerblock data">
+  <h4>Rólunk</h4>
+  <ul>
+    <li>5000 Szolnok, Tisza park 1.</li>
+    <li>Telefon: 56/514-953; 20/540-4896</li>
+    <li>E-mail: titkar@verseghy-szolnok.sulinet.hu</li>
+    <li>OM azonosító: 035992</li>
+  </ul>
+</div>
+<div class="footerblock links">
+  <h4>Linkek</h4>
+  <ul>
+    <li>Archívum</li>
+    <li>Impresszum </li>
+    <li>Oldaltérkép</li>
+  </ul>
+</div>
+<div class="footerblock subscribe">
+  <h4>Hírlevél</h4>
+  <p>Iratkozz fel hírlevelünkre, hogy értesítést kapj a téged érdeklő cikkekről!</p>
+  <input type="email" value="" name="EMAIL" class="emailInput" placeholder="email@email.com"
+         pattern="^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$" #emailInput>
+  <button class="subscribeButton" (click)="subscribeToNewsletter()" #subscribeButton>
+    <span class="buttonText" #buttonText>Feliratkozás</span>
+    <fa-icon [icon]="faCheck" hidden #checkMark></fa-icon>
```

Comment:
Please use the `*ngIf` directive or the `[hidden]` or `[attr.hidden]` object bindings (learn more about the bindings [here on stackoverflow](https://stackoverflow.com/questions/35944749/what-is-the-difference-between-parentheses-brackets-and-asterisks-in-angular2)) instead of using `hidden` or if you want to learn about animations in angular, please follow [this documentation](https://angular.io/guide/animations).